PYTHON/manifest/manifest.py

```diff
@@ -91,6 +109,8 @@ def func(name: str, input_type: str):
     outputs = []
 
     create_output = create_io(outputs)
+    if not sig.return_annotation:
```

Comment:
Shouldn't all nodes have a return annotation?

Reply:
A node that doesn't have a source handle should not contain return annotation and a node that doesn't have a target handle should not contain any parameter with DC type annotation, this is my point of view. Following this role we don't have to exclude any nodes when generating output in the manifest.